worker/go.mod

```diff
@@ -1,16 +1,13 @@
 module github.com/open-lambda/open-lambda/worker
 
 require (
-	github.com/fsouza/go-dockerclient v1.3.3
-	github.com/go-ini/ini v1.41.0 // indirect
-	github.com/gopherjs/gopherjs v0.0.0-20181103185306-d547d1d9531e // indirect
-	github.com/jtolds/gls v4.2.1+incompatible // indirect
-	github.com/minio/minio-go v6.0.12+incompatible
-	github.com/mitchellh/go-homedir v1.0.0 // indirect
-	github.com/smartystreets/assertions v0.0.0-20180927180507-b2de0cb4f26d // indirect
-	github.com/smartystreets/goconvey v0.0.0-20181108003508-044398e4856c // indirect
+	github.com/StackExchange/wmi v0.0.0-20181212234831-e0a55b97c705 // indirect
```

Comment:
why are you changing this?  Any re-sync of the vendors should be its own PR.

Reply:
Otherwise a 289-file change is just too much.

Reply:
This should not have been changed, I just happened to re-vendor all the packages... sorry about the confusion.

I have split this PR into two PRs: https://github.com/open-lambda/open-lambda/pull/54 (code only) and https://github.com/open-lambda/open-lambda/pull/55 (vendor only), and corrected the vendor sync issue in the second one.